src/debuginfo.cpp

```diff
@@ -400,25 +402,37 @@ class JuliaJITEventListener: public JITEventListener
                     assert(SectionAddrCheck == SectionLoadAddr);
                 else
                     SectionAddrCheck = SectionLoadAddr;
+#ifdef USE_ORCJIT
+                if (memmgr)
+                    SectionAddr =
+                        (uintptr_t)lookupWriteAddressFor(memmgr,
```

Comment:
@Keno BTW, any idea why the `SectionAddr` and `SectionLoadAddr` are always the same without this?